lib/rubocop/cop/lint/hash_each.rb

```diff
@@ -0,0 +1,64 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks for uses of `each_key` & `each_value` Hash methods.
+      class HashEachMethods < Cop
+        include UnusedArgument
+
+        MSG = 'Use `%s` instead of `%s.%s`.'
+
+        EACH_METHODS = [:keys, :values]
+
+        def on_block(node)
+          _block_method, args, = *node
+          @nargs = args.child_nodes.size
```

Comment:
Use `children`, not `child_nodes`. `child_nodes` filters `children` to remove any symbols and only leave `Node` objects. But that's not necessary here.